client/landing/jetpack-cloud/sections/backups/backup-activity-log/index.tsx

```diff
@@ -1,10 +1,66 @@
 /**
  * External dependencies
  */
-import React, { FunctionComponent } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+import isEqual from 'lodash/isEqual';
+import React, { FunctionComponent, useEffect } from 'react';
 
-const BackupActivityLogPage: FunctionComponent = () => {
-	return <div>{ 'Welcome to activity log.' }</div>;
+/**
+ * Internal dependencies
+ */
+import { getHttpData } from 'state/data-layer/http-data';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { requestActivityLogs, getRequestActivityLogsId } from 'state/data-getters';
+import { setFilter } from 'state/activity-log/actions';
+import ActivityCardList from 'landing/jetpack-cloud/components/activity-card-list';
+import getActivityLogFilter from 'state/selectors/get-activity-log-filter';
+
+interface Props {
+	after?: string;
+	before?: string;
+	group?: string;
+}
+
+const BackupActivityLogPage: FunctionComponent< Props > = ( { after, before, group } ) => {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const siteId = useSelector( getSelectedSiteId );
+	const filter = useSelector( state => getActivityLogFilter( state, siteId ) );
+	const logs = useSelector( () => getHttpData( getRequestActivityLogsId( siteId, filter ) ).data );
+
+	/*
+	 * When we load this page if the filter has a difference from the current filter that indicates we have navigate to the URL fresh
+	 * ( the filterbar makes modifications to the filter in state THEN navigates to the new page. )
+	 * Since we are loading this page fresh we want to "reset" the filter to what the query args tell us
+	 */
```

Comment:
In the future I would like to have a filter created here ( based on the URL ) which would be used in the request. The values for the selectors would be extracted and sent to the child component selectors ( for example the date range selector would get the before or after dates from the URL ). The selectors on change would trigger a redirect to a new URL, which in turn would create a new filter, etc. this would remove duplication from the URL & state.

The major current blocker is re-writing the `DateRangeSelector` as a controlled component, as it is currently as uncontrolled as possible 😬

Reply:
Love this idea. Sounds like we need to push until after the MVP though.